### PR TITLE
Mitre integration

### PR DESCRIPTION
**Pull request by aburt-demisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
MITRE ATT&CK pack that includes MITRE ATT&CK intel feed, new indicator, new fields, new indicator layout and mapping (classifier).

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
N/A

### CODE REVIEW

File: Packs/FeedMitreAttack/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## [Unreleased]
```
Comment (Itay4) [inline Packs/FeedMitreAttack/CHANGELOG.md:1]: lets remove this file

File: Packs/FeedMitreAttack/Classifiers/classifier-MITRE ATT&CK.json [hunk 2]
```diff
@@ -0,0 +1,676 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Classifiers/classifier-MITRE ATT&CK.json:1]: please make sure there are no whitespaces in file names

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitrealiases.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitrealiases.json:1]: please change file name to `incidentfield-mitrealiases.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitredetection.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitredetection.json:1]: please change file name to `incidentfield-mitredetection.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitredetection.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+			"id": "indicator_mitredetection",
+			"version": 1,
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitredetection.json:3]:
```suggestion
			"version": -1,
```

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitreextendedaliases.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitreextendedaliases.json:1]: please change file name to `incidentfield-mitreextendedaliases.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitreexternalreferences.json [hunk 7]
```diff
@@ -0,0 +1,85 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitreexternalreferences.json:1]: please change file name to `incidentfield-mitreexternalreferences.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitreid.json [hunk 8]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitreid.json:1]: please change file name to `incidentfield-mitreid.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitreimpacttype.json [hunk 9]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitreimpacttype.json:1]: please change file name to `incidentfield-mitreimpacttype.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitrekillchainphases.json [hunk 10]
```diff
@@ -0,0 +1,74 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitrekillchainphases.json:1]: please change file name to `incidentfield-mitrekillchainphases.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitrelabels.json [hunk 11]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitrelabels.json:1]: please change file name to `incidentfield-mitrelabels.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitrename.json [hunk 12]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitrename.json:1]: please change file name to `incidentfield-mitrename.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitrepermissionsrequired.json [hunk 13]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitrepermissionsrequired.json:1]: please change file name to `incidentfield-mitrepermissionsrequired.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitreplatforms.json [hunk 14]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitreplatforms.json:1]: please change file name to `incidentfield-mitreplatforms.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitresystemrequirements.json [hunk 15]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitresystemrequirements.json:1]: please change file name to `incidentfield-mitresystemrequirments.json`

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitreversion.json [hunk 16]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/indicator_mitreversion.json:1]: please change file name to `incidentfield-mitreversionjson`

File: Packs/FeedMitreAttack/metadata.json [hunk 17]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (Itay4) [inline Packs/FeedMitreAttack/metadata.json:1]: rename file to `pack_metadata.json` and fill in relevant json data

**Comment by aburt-demisto** [comment]
Re-opening, again..

File: Packs/FeedMitreAttack/IndicatorFields/indicator_mitrecontributors.json [hunk 18]
```diff
@@ -0,0 +1,46 @@
+{
```
[the remark this answers is not in the document]
Comment (aburt-demisto) [reply Packs/FeedMitreAttack/IndicatorFields/indicator_mitrecontributors.json:1]: You mean **indicator**field-mitrecontributors.json ??

**Pull request closed without merging**